src/main/java/com/hyperwallet/clientsdk/util/HyperwalletApiClient.java

```diff
@@ -154,6 +157,9 @@ protected void checkErrorResponse(final Response response) throws ParseException
     private void checkResponseHeader(Response response) {
         String contentTypeHeader = response.getHeader(CONTENT_TYPE_HEADER);
         String expectedContentType = isEncrypted ? VALID_JSON_JOSE_CONTENT_TYPE : VALID_JSON_CONTENT_TYPE;
+//        if (response.getResponseCode() == 302) {
```

Comment:
Do we need this commented out code?

Reply:
This is outdated, I cleaned up the comments but hadn't pushed the change